--- .htaccess	2014-08-30 14:33:07.000000000 +0300
+++ .htaccess	2014-08-30 14:33:51.000000000 +0300
@@ -109,6 +109,32 @@ DirectoryIndex index.php index.html inde
   # uncomment the following line:
   # RewriteBase /

+  ### BOOST START ###
+
+  # Allow for alt paths to be set via htaccess rules; allows for cached variants (future mobile support)
+  RewriteRule .* - [E=boostpath:normal]
+
+  # Caching for anonymous users
+  # Skip boost IF not get request OR uri has wrong dir OR cookie is set OR request came from this server OR https request
+  RewriteCond %{REQUEST_METHOD} !^(GET|HEAD)$ [OR]
+  RewriteCond %{REQUEST_URI} (^/(admin|cache|misc|modules|sites|system|openid|themes|node/add|comment/reply))|(/(edit|user|user/(login|password|register))$) [OR]
+  RewriteCond %{HTTPS} on [OR]
+  RewriteCond %{HTTP_COOKIE} DRUPAL_UID [OR]
+  RewriteCond %{ENV:REDIRECT_STATUS} 200
+  RewriteRule .* - [S=3]
+
+  # GZIP
+  RewriteCond %{HTTP:Accept-encoding} !gzip
+  RewriteRule .* - [S=1]
+  RewriteCond %{DOCUMENT_ROOT}/cache/%{ENV:boostpath}/%{HTTP_HOST}%{REQUEST_URI}_%{QUERY_STRING}\.html\.gz -s
+  RewriteRule .* cache/%{ENV:boostpath}/%{HTTP_HOST}%{REQUEST_URI}_%{QUERY_STRING}\.html\.gz [L,T=text/html,E=no-gzip:1]
+
+  # NORMAL
+  RewriteCond %{DOCUMENT_ROOT}/cache/%{ENV:boostpath}/%{HTTP_HOST}%{REQUEST_URI}_%{QUERY_STRING}\.html -s
+  RewriteRule .* cache/%{ENV:boostpath}/%{HTTP_HOST}%{REQUEST_URI}_%{QUERY_STRING}\.html [L,T=text/html]
+
+  ### BOOST END ###
+
   # Pass all requests not referring directly to files in the filesystem to
   # index.php. Clean URLs are handled in drupal_environment_initialize().
   RewriteCond %{REQUEST_FILENAME} !-f
